tajo-core/src/main/java/org/apache/tajo/worker/Task.java

```diff
@@ -135,7 +138,8 @@ public Task(String taskRunnerId,
   }
 
   public void initPlan() throws IOException {
-    plan = LogicalNodeDeserializer.deserialize(queryContext, request.getPlan());
+    QueryContextUtil.updatePythonScriptPath(systemConf, queryContext);
```

Comment:
As I mentioned, script path is immutable config. GeneralFunctionEval can get this property from OverridenConf or TajoConf. You may not need this.